fe/fe-core/src/main/java/com/starrocks/server/SharedNothingStorageVolumeMgr.java

```diff
@@ -0,0 +1,126 @@
+// Copyright 2021-present StarRocks, Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.starrocks.server;
+
+import com.google.common.base.Preconditions;
+import com.staros.util.LockCloseable;
+import com.starrocks.common.AlreadyExistsException;
+import com.starrocks.common.AnalysisException;
+import com.starrocks.storagevolume.StorageVolume;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+
+public class SharedNothingStorageVolumeMgr extends StorageVolumeMgr {
+    private Map<String, StorageVolume> nameToSV = new HashMap<>();
```

Comment:
// TODO: add persistency ?

Reply:
OK, I will add serialized name in this PR.

Reply:
done